compiler/rustc_middle/src/ty/print/pretty.rs

```diff
@@ -1710,6 +1710,10 @@ pub trait PrettyPrinter<'tcx>: Printer<'tcx> + fmt::Write {
             ty::Bool if int == ScalarInt::FALSE => p!("false"),
             ty::Bool if int == ScalarInt::TRUE => p!("true"),
             // Float
+            ty::Float(ty::FloatTy::F16) => {
```

Comment:
Should this be made a nested match so we get exhaustiveness checking?

Reply:
Sure, opened https://github.com/rust-lang/rust/pull/127224